deployment/conf/.templates/start_workspace.sh.templ

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+cd /kb/deployment
+
+# Values for port, threads, min/max memory are populated into source template from environment vars
+# the defaults are based on
+bin/glassfish_administer_service.py --verbose --admin $GLASSFISH/bin/asadmin --domain Workspace \
```

Comment:
I assume you want an --instanceport argument here

Reply:
Not really, there is a default in the glassfish admin script that just uses the normal port 8080, which is the hard coded default in glassfish.

Reply:
Okay, forgot to commit that update